src/packages/@ncigdc/containers/ConsequencesTable.js

```diff
@@ -131,6 +136,7 @@ const ConsequencesTableComponent = compose(
       style={{ width: '100%', minWidth: 450 }}
       data={dataRows}
       prefix={paginationPrefix}
+      entityName={'entries'}
```

Comment:
I would keep "Show 10 entries" as it's very standard in all our tables and display `Showing 1 - 10 of 27` as suggested to avoid the debate about consequences/transcripts. Thanks!